### DISCUSSION TITLE
Sunbird video player  implementation in native android webview

### DISCUSSION THREAD

**Discussion by swayangjit** [Sunbird-Knowlg/Community#144] [Bugs]
'Sunbird Video Player' on mobile app(built using Android/Kotlin). 

Video plays ONLY in 'Landscape' mode. Our requirement is to play video in 'Portrait' mode as well.
The video player controls are NOT functioning. For example, dragging the video tracker forward will make the video play from the beginning instead of its new position.
The '[ ]' icon at the bottom-right does not work.
A sample implementation is available in [this comment](https://github.com/orgs/Sunbird-Ed/discussions/569#discussioncomment-6422670)

@HarishGangula

**Comment by HarishGangula** [comment]
@swayangjit  If the content doesn't have the streaming URL then seek behavior like that only in video player and it plays both in landscape and portrait mode,  as player it doesn't have any limitation based on viewport size width it renders accrodingly.

**Reply by pallakartheekreddy** [reply]
@swayangjit As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
